packages/svelte2tsx/src/nodes/ComponentEvents.ts

```diff
@@ -41,6 +42,52 @@ export class ComponentEventsFromInterface extends ComponentEvents {
         return map;
     }
 
+    private getName(prop: ts.PropertyName) {
+        if (ts.isIdentifier(prop) || ts.isStringLiteral(prop)) {
```

Comment:
Also NumberLiteral

Reply:
According to the Svelte typings, only `string` is possible for the events: https://github.com/sveltejs/svelte/blob/master/src/runtime/internal/lifecycle.ts#L32

Reply:
If it's official now then is fine. I just tried it in REPL and find it possible as a number XD.

Reply:
I think it will be stringified, so the correct typing would be to only allow `string`.